coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetViewHelperServiceImpl.java

```diff
@@ -205,4 +213,16 @@ public void setKcBusinessRulesEngine(KcBusinessRulesEngine kcBusinessRulesEngine
 		this.kcBusinessRulesEngine = kcBusinessRulesEngine;
 	}
     
+	public boolean isBudgetMarkedForSubmission() {
+		ProposalBudgetForm form = (ProposalBudgetForm)ViewLifecycle.getModel();
```

Comment:
I'm ok with this, but I dislike the useing ViewLifecycle. Global usages have plagued us. I'd prefer if you pass in the budget using SpEL @{budget}.